website/content/docs/nia/release-notes/0-5-0.mdx

```diff
@@ -0,0 +1,215 @@
+---
+layout: docs
+page_title: CTS v0.5.0
+description: >-
+  Consul-Terraform-Sync Configuration Deprecations for v0.5.0
+---
+
+# Configuration Deprecations in v0.5
+
+We introduced a number of configuration deprecations in Consul-Terraform-Sync (CTS) v0.5. Many of these deprecations are part of an effort to streamline CTS task configuration and reduce the inconsistencies. As CTS supported more use-cases, the inconsistencies grew and required a lot of documentation to explain.
+
+This documentation outlines these deprecations and examples on how to upgrade your configuration for v0.5. The deprecated configurations will not be removed until v0.7 and later. We plan to also release a tool to automate upgrading your configuration to hopefully make the upgrade quicker and smoother.
```

Comment:
```suggestion
This documentation outlines the deprecations and examples on how to upgrade your configuration for v0.5. The deprecated configurations will not be removed until v0.7 and later. We plan to also release a tool to automate upgrading your configurations to make the upgrade quicker and smoother.
```
I removed hope. We want to sound confident.  We know the tool will help because you are all awesome 😄